modules/sdk-coin-apt/src/lib/constants.ts

```diff
@@ -9,6 +9,8 @@ export const SECONDS_PER_WEEK = 7 * 24 * 60 * 60; // Days * Hours * Minutes * Se
 export const APTOS_ACCOUNT_MODULE = 'aptos_account';
 export const FUNGIBLE_ASSET_MODULE = 'primary_fungible_store';
 
-export const FUNGIBLE_ASSET_FUNCTION = '0x1::primary_fungible_store::transfer';
+export const FUNGIBLE_ASSET_TRANSFER_FUNCTION = '0x1::primary_fungible_store::transfer';
+export const COIN_TRANSFER_FUNCTION = '0x1::aptos_account::transfer_coins';
 
-export const FUNGIBLE_ASSET = '0x1::fungible_asset::Metadata';
+export const FUNGIBLE_ASSET_TYPE_ARGUMENT = '0x1::fungible_asset::Metadata';
```

Comment:
Tomorrow there may be other types of transactions like fungible asset staking etc.
The type argument may be different for it.
```suggestion
export const FUNGIBLE_ASSET_TRANSFER_TYPE_ARGUMENT = '0x1::fungible_asset::Metadata';
```